dao-contracts/src/voting/governance_voting.rs

```diff
@@ -399,49 +383,30 @@ impl GovernanceVoting {
     }
 
     fn perform_action(&self, voting: &Voting) {
```

Comment:
What would you say to implement it like that https://github.com/make-software/dao-contracts/commit/0ddef6b7e6aaa797c7aefa1329aa0c53ce09ad27#diff-517bedb2a801ee5ee0f3ad949832a70ff9985e63a37664072d4291b67417dbb3?
It would work as `Erc721Receiver` (see dao-erc721), so each voting contract implements a trait (eg. VotingCallback).
The signature would be like `on_voting_finished(&self, args: RuntimeArgs)`